tests/citeproc-test.py

```diff
@@ -22,6 +22,7 @@
                                      'test-suite'))
 TEST_PARSER_PATH = os.path.join(CITEPROC_TEST_PATH, 'processor.py')
 TESTS_PATH = os.path.join(CITEPROC_TEST_PATH, 'processor-tests', 'humans')
+LOCAL_TESTS_PATH = os.path.join(os.path.dirname(__file__), "local")
```

Comment:
what meaning did you imply by "local"? should it be may be "resources" or "files"?

in any case -- would also need to tune up `MANIFEST.in` to include .txt files I guess.